Fighting Game Builder/build.py

```diff
@@ -79,67 +101,38 @@
                 })
 
             match_name = mode.get("match_name", "match")
-            if "characters" in mode:
-                characters = [f"{mode_name} - {char}" for char in mode['characters']]
-                for char in characters:
-                    j_items.append({
-                        "name": char,
-                        "category": [f"{mode_name} {playable_alias}"],
-                        "progression": True,
-                        "count": 1
-                    })
-            elif "characters" in y_data:
-                characters = y_data['characters']
-            else:
-                raise ValueError(f"Character set not defined for mode {mode_name}")
-            for char in characters:
+            # if there are mode specific characters
+            mode_characters = mode.get("characters", [])
+            # remove duplicates from global characters
+            mode_characters = set(mode_characters).difference(global_characters)
```

Comment:
What is this line trying to do? I'm not registering the logic

Reply:
if a character was set as a global unlock it shouldnt also be able to be a mode unlock